main.js

```diff
@@ -0,0 +1,471 @@
+$(document).ready(startConnectFour);
+
+var gameBoardArray = [
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0]
+];
+
+var vectors = [
+  { y1: 0, x1: -1, y2: 0, x2: 1 }, //left and right
+  { y1: 1, x1: 0, y2: -1, x2: 0 }, //up and down
+  { y1: 1, x1: 1, y2: -1, x2: -1 }, //down right and up left
+  { y1: 1, x1: -1, y2: -1, x2: 1 } //down left and up right
+];
+
+var crossVector = [
+  { y1: 0, x1: -1, y2: 0, x2: 1 }, //left and right
+  { y1: 1, x1: 0, y2: -1, x2: 0 } //up and down
+];
+
+var LVector = [{ y1: 0, x1: 1, y2: -1, x2: 0 }];
+
+var player = 0;
+var symbols = [];
+
+var powerUps = {
+  moon: 0,
+  sun: 0
+};
+
+function togglePlayer() {
+  player = 1 - player;
+}
+
+checkForWin(0, 3, 1);
+
+function startConnectFour() {
+  console.log("start Connect Four");
+  // createCells(7, 7);
+  addGameHandlers();
+  clickStartButton();
+  addPlanetHandler();
+}
+
+function assignPlayer() {
+  if (symbols.length >= 2) {
+    return;
+  }
+
+  if ($(event.target).hasClass("ifImageChosen")) {
+    return;
+  }
+  var symbol = $(this).attr("src");
+  if (player === 0) {
+    symbols[0] = symbol;
+  } else {
+    symbols[1] = symbol;
+  }
+
+  $(this).addClass("ifImageChosen");
+
+  togglePlayer();
+  changeTextonModal();
+}
+
+function changeTextonModal() {
+  if (symbols.length === 1) {
+    $(".selectPlayer").text("Select the Planetary Body for Player 2");
```

Comment:
When doing dom manipulation like this you should preferably be making use of ids rather than classes. Its faster for jquery to find id elements on the dom than anything else.